### PR TITLE
updating cwlogs collection docs

### PR DESCRIPTION
**Pull request by himsharma01**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Review by npande — APPROVED** [review]
Technical content LGTM.

File: docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-cloudformation-template.md [hunk 1]
```diff
@@ -224,6 +229,16 @@ If you only need to collect logs from a few additional CloudWatch Log groups, yo
     <br/><img src={useBaseUrl('img/send-data/create-destination.png')} style={{border: '1px solid gray'}} alt="create-destination" width="700"/>
 1. (Optional) In the **Test Pattern** section, select the log data to test, then click **Test pattern**. If test results look fine, then click **Start Streaming**.<br/><img src={useBaseUrl('img/send-data/test-pattern.png')} style={{border: '1px solid gray'}} alt="test-pattern" width="700"/>
 
+:::note
+Sample cloudWatch Log ingested for Amazon RDS postgresSQL using Lambda Log forwarder with ```IncludeLogGroupInfo``` set to ```false``` .
```
Comment (jpipkin1) [inline docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-cloudformation-template.md:233]:
```suggestion
Sample cloudWatch Log ingested for Amazon RDS postgreSQL using Lambda Log forwarder with ```IncludeLogGroupInfo``` set to ```false``` . 
```

File: docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-lambda-function.md [hunk 2]
```diff
@@ -72,3 +77,13 @@ If you have more than one CloudWatch Log Group to assign to your Lambda function
 1. Add a **Filter Name** to your trigger.
 1. (Optional) Add a **Filter Pattern** to your trigger. For information about AWS filter patterns, see [Filter and Pattern Syntax](http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html) in the AWS documentation.
 1. Click **Add** to add the trigger to your Lambda function.
+
+:::note
+Sample cloudWatch Log ingested for Amazon RDS postgresSQL using Lambda Log forwarder.
```
Comment (jpipkin1) [inline docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-lambda-function.md:82]:
```suggestion
Sample cloudWatch Log ingested for Amazon RDS postgreSQL using Lambda Log forwarder. 
```

File: docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-cloudformation-template.md [hunk 3]
```diff
@@ -185,6 +185,11 @@ If you do not want the `SumoCWSpilloverAlarm` alarm to be created, remove the de
 
 After few minutes you will see `CREATE_COMPLETE` in the Status column.
 
+:::note
+This solution only ingests log messages to sumo logic. If you want to include log levels (such as INFO, DEBUG,etc) to sumo logic, please find and update the regex in [lambda code](https://github.com/SumoLogic/sumologic-aws-lambda/blob/main/cloudwatchlogs-with-dlq/cloudwatchlogs_lambda.js#L14) deployed in your AWS account with
```
Comment (jpipkin1) [inline docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-cloudformation-template.md:189]:
```suggestion
This solution only ingests log messages to Sumo Logic. If you want to include log levels (such as INFO, DEBUG, etc.) to Sumo Logic, find and update the regex in [Lambda code](https://github.com/SumoLogic/sumologic-aws-lambda/blob/main/cloudwatchlogs-with-dlq/cloudwatchlogs_lambda.js#L14) deployed in your AWS account with 
```

File: docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-cloudformation-template.md [hunk 4]
```diff
@@ -224,6 +229,16 @@ If you only need to collect logs from a few additional CloudWatch Log groups, yo
     <br/><img src={useBaseUrl('img/send-data/create-destination.png')} style={{border: '1px solid gray'}} alt="create-destination" width="700"/>
 1. (Optional) In the **Test Pattern** section, select the log data to test, then click **Test pattern**. If test results look fine, then click **Start Streaming**.<br/><img src={useBaseUrl('img/send-data/test-pattern.png')} style={{border: '1px solid gray'}} alt="test-pattern" width="700"/>
 
+:::note
+Sample cloudWatch Log ingested for Amazon RDS postgreSQL using Lambda Log forwarder with ```IncludeLogGroupInfo``` set to ```false``` .
```
Comment (jpipkin1) [inline docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-cloudformation-template.md:233]:
```suggestion
Sample CloudWatch log ingested for Amazon RDS PostgreSQL using Lambda log forwarder with ```IncludeLogGroupInfo``` set to ```false``` . 
```

File: docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-lambda-function.md [hunk 5]
```diff
@@ -54,6 +54,11 @@ To add an Amazon Lambda function:
     * **SOURCE_NAME_OVERRIDE (Optional)**. Override `_sourceName` value configured for the HTTP source.
 1. Click **Create Function.** A success message will be displayed.<br/><img src={useBaseUrl('img/send-data/success.png')} style={{border: '1px solid gray'}} alt="success" width="700"/>
 
+:::note
+This solution only ingests log messages to sumo logic. If you want to include log levels (such as INFO, DEBUG,etc) to sumo logic, please find and update the regex in [lambda code](https://github.com/SumoLogic/sumologic-aws-lambda/blob/main/cloudwatchlogs/cloudwatchlogs_lambda.js#L32) deployed in your AWS account with
```
Comment (jpipkin1) [inline docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-lambda-function.md:58]:
```suggestion
This solution only ingests log messages to Sumo Logic. If you want to include log levels (such as INFO, DEBUG, etc.) to Sumo Logic, find and update the regex in [Lambda code](https://github.com/SumoLogic/sumologic-aws-lambda/blob/main/cloudwatchlogs/cloudwatchlogs_lambda.js#L32) deployed in your AWS account with 
```

File: docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-lambda-function.md [hunk 6]
```diff
@@ -72,3 +77,13 @@ If you have more than one CloudWatch Log Group to assign to your Lambda function
 1. Add a **Filter Name** to your trigger.
 1. (Optional) Add a **Filter Pattern** to your trigger. For information about AWS filter patterns, see [Filter and Pattern Syntax](http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html) in the AWS documentation.
 1. Click **Add** to add the trigger to your Lambda function.
+
+:::note
+Sample cloudWatch Log ingested for Amazon RDS postgreSQL using Lambda Log forwarder.
```
Comment (jpipkin1) [inline docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-lambda-function.md:82]:
```suggestion
Sample CloudWatch log ingested for Amazon RDS PostgreSQL using Lambda log forwarder. 
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@himsharma01, I found a few things to update.

**Comment by himsharma01** [comment]
> @himsharma01, I found a few things to update.

Thanks for your review @jpipkin1 . Changes looks good to me.

**Review by jpipkin1 — APPROVED** [review]

**Comment by jpipkin1** [comment]
@himsharma01, I'll go ahead and merge this PR.

**Pull request merged**